### ISSUE TITLE
Feature: support inline HTML in flutter web

### ISSUE THREAD

**Issue by felipecrs** [flutter/flutter_markdown#345]
**Is your feature request related to a problem? Please describe.**
No, it's in fact due to the limitation described in the README.

**Describe a possible solution.**
I really don't know, but I wonder if this could be supported at least in flutter web, which can fallback to display HTMLs using the browser renderer.

**Describe possible alternatives.**
If a HTML2Dart translator (even a very simple one) could be implemented, this feature could be portable to other platforms as well.

**Additional context.**
```dart
body: Center(
  child: SafeArea(
    child: Markdown(data: "Text **bold** <br></br> _Italic_"),
  ),
),
```

![image](https://user-images.githubusercontent.com/29582865/110561191-022a5080-8126-11eb-9c22-688a01f5145a.png)

**Comment by domesticmouse** [comment]
I suspect for the use case of rendering markdown in a browser that using a JavaScript markdown parser in a plugin would make more sense. Flutter's text rendering on desktop browsers is via CanvasKit, not HTML, by default.

**Comment by felipecrs** [comment]
It may be a dumb question but, if the [markdown](https://pub.dev/packages/markdown) library converts Markdown to HTML, and this library (flutter_markdown) takes from it, why can't the inline HTML work just as the generated HTML from markdown does?

**Comment by domesticmouse** [comment]
I believe the place to start researching is [here](https://github.com/flutter/flutter_markdown/blob/master/lib/src/builder.dart#L93).

Tl;dr: The `markdown` library does two things, first it parses markdown into an internal tree representation, and second it turns that internal representation into HTML. Your assumption that `flutter_markdown` consumes HTML from `markdown` is incorrect, we actually consume the internal tree representation.

**Comment by felipecrs** [comment]
Fair. Thanks for the answer.

**Comment by felipecrs** [comment]
Ok, so searching more about the HTML2Dart alternative that I proposed, I found [flutter_html](https://pub.dev/packages/flutter_html), which is able to render HTML. Could `flutter_markdown` leverage `flutter_html`?

A third option would be to use the `markdown` library to convert Markdown to HTML and then use `flutter_html` to render it on the screen.

**Comment by domesticmouse** [comment]
I'd suggest spiking that out as a new project, using `markdown` to feed into `flutter_html` and seeing what the speed and memory impacts are. I suspect it would wind up being a larger binary, and probably slower to render as there is more code in the pipeline.
